src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -60,12 +60,14 @@ export interface NavBarPropsType {
   hasBackButton?: boolean;
   withSchedule?: boolean;
   withPhotobooth?: boolean;
+  hideLoginButton?: boolean;
 }
 
 export const NavBar: React.FC<NavBarPropsType> = ({
   hasBackButton,
   withSchedule,
   withPhotobooth,
+  hideLoginButton,
```

Comment:
Should this have a default value?

Reply:
Yes please, also, maybe in the vein of `hasLoginButton` or `withHiddenLoginButton` instead of a verb. Is it a function?